storage/container_storage_interface/persistent-storage-csi-cloning.adoc

```diff
@@ -5,7 +5,7 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-This document describes how to use volume cloning with supported Container Storage Interface (CSI) drivers to protect against data loss in {product-title}. Familiarity with xref:../../storage/understanding-persistent-storage.adoc#persistent-volumes_understanding-persistent-storage[persistent volumes] is suggested.
+Volume cloning duplicates an existing persistent volume to protect against data loss in {product-title}. This feature is only available with supported Container Storage Interface (CSI) drivers. You should be familiar with xref:../../storage/understanding-persistent-storage.adoc#persistent-volumes_understanding-persistent-storage[persistent volumes] before you begin.
```

Comment:
"before you begin" what? Before you provision a CSI volume clone?

Reply:
Yes. I've changed to say "before you provision a CSI volume clone."